lib/elixir/test/elixir/application_test.exs

```diff
@@ -21,6 +21,20 @@ defmodule ApplicationTest do
     assert Application.get_env(:elixir, :unknown, :default) == :default
   end
 
+  test "application information" do
+    started = Application.started_applications
+    assert started |> is_list
```

Comment:
Hah! The programming equivalent of dad saying he'll turn the car around if you don't shut up in the back :)